src/docs/feature-flags/index.mdx

```diff
@@ -11,8 +11,8 @@ You can find a list of features available by looking at
 `sentry/features/__init__.py`. They're declared on the `FeatureManager` like so:
 
 ```python
-# Don't set entity_feature, or set it to False if you don't plan to use Flagr
-default_manager.add("organizations:onboarding", OrganizationFeature, entity_feature=True)
+# pass FeatureHandlerStrategy.REMOTE to use flagr, or FeatureHandlerStrategy.INTERNAL if you don't plan to use flagr.
```

Comment:
On widescreens, there's a scroll overflow:

<img width="760" alt="Screenshot 2023-03-20 at 2 01 02 PM" src="https://user-images.githubusercontent.com/139499/226427314-62380c9b-8143-470b-b1f4-11b03fbe4a9f.png">

Maybe consider adding an example for `FeatureHandlerStrategy.INTERNAL` as well